File: IMS/IMS.DAL/Enums/Enums.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace IMS.DAL.Enums
{
    public enum Role
    {
        Unassigned,
        Admin,
        HRManager,
        Mentor,
        Intern
    }

    public enum Status
    { 
        Unassigned,
        ToDo,   
        InProgress,
        PullRequest,
        Done
    }
}

File: Frontend/ims-frontend/src/entities/ims/enums.ts
export enum Role {
    HumanResourcesManager = 2,
    Mentor = 3,
    Intern = 4
}

export enum TicketStatus {
    Unassigned = 0,
    ToDo = 1,
    InProgress = 2,
    PullRequest = 3,
    Done = 4
}

export enum InternshipStatus{
    NotStarted = 0,
    Ongoing = 1,
    Completed = 2,
    Cancelled = 3
}

export enum UserSortingParameter{
    None = 0,
    AscendingId = 1,
    DescendingId = 2,
    AscendingFirstName = 3,
    DescendingFirstName = 4,
    AscendingLastName = 5,
    DescendingLastName = 6,
    AscendingRole = 7,
    DescendingRole = 8
}
